definitions/test/e2e/enhanced-confidentiality.spec.ts

```diff
@@ -22,7 +22,7 @@ test.describe("Enhanced confidentiality test", async() => {
         await createBundleSteps.triggerBundleForConfidentialCase();
     });
 
-    test("Refuse - confidentiality request for a party on a case", {tag: ['@nightly-pipeline']}, async({ uploadResponseSteps, enhancedConfidentialitySteps }) => {
+    test("Refuse - confidentiality request for a party on a case", {tag: ['@preview-regression','@nightly-pipeline']}, async({ uploadResponseSteps, enhancedConfidentialitySteps }) => {
```

Comment:
do you need to mention the preview-regression tag in the readme too?

Reply:
fine, but its still not clear in the readme that its the test-suite:<labels> you add in github which picks up specific tests according to <labels>